Dockerfile

```diff
@@ -10,11 +10,8 @@ RUN mkdir -p /srv/configurable-http-proxy
 COPY . /srv/configurable-http-proxy
 WORKDIR /srv/configurable-http-proxy
 
-# Install configurable-http-proxy, then automatically install compatible updates
-# to vulnerable dependencies, and finally uninstall npm which isn't needed.
-RUN npm install -g --production \
- && npm audit fix \
- && npm uninstall -g npm
+# Install configurable-http-proxy
+RUN npm install -g --production
```

Comment:
```suggestion
RUN npm ci --production
```

I wanted us to start using `npm ci` to work against package-lock.json, but we can not use `-g` with `npm ci`, so that need to be resolved somehow as the `chp-docker-entrypoint` script assumes `configurable-http-proxy` to be on path.